### ISSUE TITLE
more folios than canvasses crashes map_folios

### ISSUE THREAD

**Issue by eliwaksbaum** [DDMAL/cantus#415]
```
if mapped_folios == 0:
            for idx, folio in enumerate(folios):
                uris_objs[idx]["folio"] = folio
```

in implementing this default mapping, we assume there at least as many canvasses as folios, which is not always the case, resulting in index out of bounds errors

**Comment by eliwaksbaum** [comment]
```
if mapped_folios == 0 and len(uris_objs) >= len(folios):
            for idx, folio in enumerate(folios):
                uris_objs[idx]["folio"] = folio
```

seems to do the trick. just no default values are assigned. do we even want the defaults in a regular case? They hardly ever seem to be on the ball

**Comment by napulen** [comment]
> do we even want the defaults in a regular case? They hardly ever seem to be on the ball

The rationale for this is that it is easier to use the `Up` and `Down` buttons of the interface to move the pre-located strings around, than to type the folio numbers from scratch.

**Comment by napulen** [comment]
Agreed that in the case of this weird situation (more folios than images), it makes sense to not prepopulate anything. We fully delegate to the user in that case.
